app/models/form.rb

```diff
@@ -6,6 +6,7 @@ class Form < ActiveRecord::Base
   accepts_nested_attributes_for :fields
 
   validates :name, presence: true, uniqueness: true
+  validates :reply_to, presence: true
```

Comment:
Since the mailer has a default reply_to value, I don't think we need to validate its presence in the form model.